previews/anchor/table_component_preview.rb

```diff
@@ -13,19 +13,36 @@ def [](key)
       end
     end
 
-    def playground
-      render Anchor::TableComponent.new(data:, caption: "Demo table") do |table|
-        table.with_column(header: "Id", value: :id)
+    # @param caption text
+    # @param paginate toggle
+    # @param rows_link_to text
+    # @param sortable toggle
+    def playground(
+      caption: "Demo table",
+      paginate: false,
+      rows_link_to: "a_link/",
```

Comment:
Let's add `#` so clicking in preview doesn't take you to an inexistent path (same below):
```suggestion
      rows_link_to: "#a_link/",
```